README.md

```diff
@@ -79,6 +79,9 @@ best_it_commercetools_filter:
 
     # Optional used translation domain (default "messages")
     translation_domain: app
+
+    # Cache life time
+    cache_life_time: 86400
```

Comment:
Documentation must provide its default values (currently 86400 vs. 0) use http://symfony.com/blog/new-in-symfony-2-4-dumping-the-default-configuration-for-an-extension.
currently if installing this bundle via readme file there is no clue about that nothing is cached by default.